src/main/scala/cromwell/engine/finalcall/FinalCall.scala

```diff
@@ -1,32 +1,64 @@
 package cromwell.engine.finalcall
 
 import cromwell.engine._
-import cromwell.engine.backend.ExecutionHandle
-import cromwell.engine.workflow.ExecutionStoreKey
-import wdl4s.Scope
+import cromwell.engine.backend.{SuccessfulFinalCallExecution, ExecutionHandle}
+import cromwell.engine.workflow.{FinalCallKey, ExecutionStoreKey}
+import cromwell.webservice.WorkflowMetadataResponse
+import wdl4s.{Workflow, Scope}
 
 import scala.concurrent.{ExecutionContext, Future}
 
 object FinalCall {
 
+  private val finalCallCompanions: Traversable[FinalCallCompanion[_ <: FinalCall]] = List(
```

Comment:
bleh, the danger of my war on Seq. You all start using traversable and iterable :)